.github/MAINTAINING.md

```diff
@@ -0,0 +1,88 @@
+# Maintaining the Terraform AWS Provider
+
+<!-- TOC depthFrom:2 -->
+
+- [Pull Requests](#pull-requests)
+    - [Pull Request Review Process](#pull-request-review-process)
+        - [Dependency Updates](#dependency-updates)
+            - [AWS Go SDK Updates](#aws-go-sdk-updates)
+            - [Terraform Updates](#terraform-updates)
+    - [Pull Request Merge Process](#pull-request-merge-process)
+- [Release Process](#release-process)
+
+<!-- /TOC -->
+
+## Pull Requests
+
+### Pull Request Review Process
+
+Notes for each type of pull request are (or will be) available in subsections below.
+
+- If you are planning on owning the pull request through merge/closure, assign it to yourself
+- Add `bug`, `enhancement`, `new-data-source`, `new-resource`, or `technical-debt` labels to match expectations from change
+- Review the contents of the pull request and ensure the change follows the relevant section of the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#checklists-for-contribution)
+- If the change is not acceptable, leave a long form comment about the reasoning and close the pull request
+- If the change is acceptable with modifications, leave a pull request review marked using the `Request Changes` option (for maintainer pull requests with minor modification requests, giving feedback with the `Approve` option is recommended so they do not need to wait for another round of review)
+- If the author is unresponsive for changes (generally we give two weeks), determine importance and level of effort to finish the pull request yourself including their commits or close the pull request
+- Run relevant acceptance testing (locally or in TeamCity) and ensure no new failures are being introduced
```

Comment:
Can this link to any additional documentation on running acceptance tests? When does it make sense to use TeamCity?

Reply:
Also -- where does a maintainer look to decide which regions or clouds the tests should be run against?

Reply:
Hmm. We can likely link to the [Contributing Guide section for running acceptance testing](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#running-an-acceptance-test). The TeamCity documentation and access to it are considered internal at the moment. I'm not sure we have plans to change either of those. TeamCity is generally helpful when there is a lot to test or those tests will potentially take a long time. The heuristics for those are fuzzy.

Reply:
Updated this line to include link to Contributing Guide as well as mentioning AWS Commercial and GovCloud (US) although GovCloud (US) is not a strict requirement at the moment.